### ISSUE TITLE
template partial specialization is not more specialized than the primary template

### ISSUE THREAD

**Issue by starplanet** [HIT-SCIR/pyltp#126]
## *What* is affected by this bug?
Can't install pyltp on Mac OS High Sierra 10.13.1.

## *When* does this occur?
`pip install pyltp`

## *Where* on the code does it happen?
It will report following problem:
```
ltp/thirdparty/eigen/unsupported/Eigen/CXX11/src/Tensor/TensorStorage.h:39:7: error: class template partial specialization is not more specialized than the primary template [-Winvalid-partial-specialization]
    class TensorStorage<T, FixedDimensions, Options_>
          ^
    ltp/thirdparty/eigen/unsupported/Eigen/CXX11/src/Tensor/TensorStorage.h:34:63: note: template is declared here
    template<typename T, typename Dimensions, int Options_> class TensorStorage;
                                                                  ^
    1 error generated.
    error: command 'clang++' failed with exit status 1
```

## *How* do we replicate the issue?
`pip install pyltp`

## Your environment information
Mac OS High Sierra 10.13.1
Python 3.6.1
pip 9.0.1
pyltp 0.2.0

## Expected behavior (i.e. solution)
install successfully

## Other Comments
I can add '-Wno-invalid-partial-specialization' option to setup.py to resolve this problem. I hope this fix can merge into source code.

**Comment by liu946** [comment]
The bug has been solved on the c++ main repo. Pyltp will fix this after #127 merged.

**Comment by liu946** [comment]
Sloved.
